new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiDomain.java

```diff
@@ -1074,6 +1075,7 @@ private static void collectAppAvaiability(
       
       if (count == 0) {
         logger.info("XXXXXXXXXXX: application not available XXXXXXXX");
+        failed = true;
```

Comment:
This is an odd programming style. Why not use break?

Reply:
Related question... why aren't you using our ordinary retry pattern?

Reply:
> This is an odd programming style. Why not use break?

Yes, actually I can just exit the loop with break.

Reply:
> Related question... why aren't you using our ordinary retry pattern?

Did you mean using ConditionFactory?

Reply:
The pattern that we use in the Unit test cases are easier for a simple condition retries. Here we have additional operation to perform. So I did not think that was a better pattern for this.